JAICore/jaicore-ml/src/main/java/ai/libs/jaicore/ml/core/dataset/serialization/ArffDatasetAdapter.java

```diff
@@ -40,12 +40,17 @@
 import ai.libs.jaicore.ml.core.dataset.SparseInstance;
 import ai.libs.jaicore.ml.core.dataset.schema.LabeledInstanceSchema;
 import ai.libs.jaicore.ml.core.dataset.schema.attribute.IntBasedCategoricalAttribute;
+import ai.libs.jaicore.ml.core.dataset.schema.attribute.MultidimensionalAttribute2d;
+import ai.libs.jaicore.ml.core.dataset.schema.attribute.MultidimensionalAttribute3d;
 import ai.libs.jaicore.ml.core.dataset.schema.attribute.NumericAttribute;
 import ai.libs.jaicore.ml.core.dataset.schema.attribute.StringAttribute;
 import ai.libs.jaicore.ml.core.dataset.serialization.arff.EArffAttributeType;
 import ai.libs.jaicore.ml.core.dataset.serialization.arff.EArffItem;
 import ai.libs.jaicore.ml.pdm.dataset.SensorTimeSeriesAttribute;
 
+/**
+ * an Adapter that reads an ARFF File and parses it into Attribute Value Objects by using instances of Attribute

```

Comment:
"Handles dataset files in the arff format" 
@link https://www.cs.waikato.ac.nz/ml/weka/arff.html